t/plugin/prometheus3.t

```diff
@@ -90,3 +90,106 @@ passed
 [200, 200]
 --- no_error_log
 [error]
+
+
+
+=== TEST 3: apisix_batch_process_entries, mess with global rules
```

Comment:
Could you use the full file in https://github.com/apache/apisix/pull/6047#issuecomment-1008495244 instead of part of it?

Reply:
OK